.sonarcloud.properties

```diff
@@ -0,0 +1,3 @@
+sonar.python.version = 3.8, 3.9, 3.10, 3.11
```

Comment:
Why 3.8 in here?

Reply:
Matches https://github.com/mewler/mewbot/blob/mainline/setup.py#L30
I think 3.8 is what is specified everywhere as the requirement, although the linters do currently run (only) on 3.9